index.js

```diff
@@ -28,30 +23,39 @@ app.use(
 
 const PORT = 3000;
 app.listen(PORT, () => {
-  console.log("SERVER IS UP ON PORT:", PORT);
+  console.log('SERVER IS UP ON PORT:', PORT);
 });
 
 async function executeQuery(query) {
   return new Promise((resolve, reject) => {
+    // Create a new connection pool
+    const pool = new Pool({
+      user: process.env.DB_USER,
+      host: 'localhost',
+      database: process.env.DB_NAME,
+      password: process.env.DB_PASSWORD,
+      port: process.env.DB_PORT,
+    });
```

Comment:
is this opening the connection on readonly mode? if not, is there a way to specify that connection role?